<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TuxCut-5.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/tuxcut.png</normaloff>:/images/images/tuxcut.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cboxProtection">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Protection Mode</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Network Interfaces :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboIfaces"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_changemac">
          <property name="text">
           <string>Change MAC</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/mac.png</normaloff>:/images/images/mac.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Current MAC Address :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_mac">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tblHosts">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>IP Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MAC Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hostname</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Refresh </string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/refresh.png</normaloff>:/images/images/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cut">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Cut</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_resume">
        <property name="text">
         <string>Resume</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/resume.png</normaloff>:/images/images/resume.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_resume_all">
        <property name="text">
         <string>Resume All</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/resume.png</normaloff>:/images/images/resume.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_about">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/about.png</normaloff>:/images/images/about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menu_Language"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Utilities">
    <property name="title">
     <string>&amp;Utilities</string>
    </property>
    <widget class="QMenu" name="menuSpeed_Limiter">
     <property name="title">
      <string>Speed Limiter</string>
     </property>
    </widget>
    <addaction name="menuSpeed_Limiter"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Utilities"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_protection_changes()</slot>
  <slot>on_refresh_clicked()</slot>
  <slot>on_cut_clicked()</slot>
  <slot>on_resume_clicked()</slot>
  <slot>on_quit_triggered()</slot>
  <slot>change_mac()</slot>
  <slot>on_about_clicked()</slot>
  <slot>resume_all()</slot>
  <slot>arabic_selected()</slot>
  <slot>english_selected()</slot>
  <slot>limit_speed()</slot>
  <slot>unlimit_speed()</slot>
  <slot>about_qt()</slot>
 </slots>
</ui>
